src/main/webapp/app/shared/metis/post/post.component.html

```diff
@@ -60,30 +62,40 @@
                     (userReferenceClicked)="onUserReferenceClicked($event)"
                     (channelReferenceClicked)="onChannelReferenceClicked($event)"
                 />
+            } @else if (isDeleted) {
+                <span class="d-inline-flex align-items-center">
+                    <span class="text-secondary" jhiTranslate="artemisApp.metis.post.deletedContent" [translateValues]="{ progress: deleteTimerInSeconds }"></span>
+                    <button class="btn btn-outline-primary btn-sm ms-2 position-relative">
+                        <span class="post-delete-button-label" jhiTranslate="artemisApp.metis.post.undoDelete" (click)="onDeleteEvent(false)"></span>
+                        <span class="post-delete-button-background"></span>
+                    </button>
+                </span>
```

Comment:
Can you move this into the jhi-posting-content to reduce duplication?

Reply:
Moved the delete prompt into jhi-posting-content component in this [commit](https://github.com/ls1intum/Artemis/pull/9624/commits/82a35634c5635a96949b32342c0bee087f8da25a).